app/components/TxHistory/TxHistory.js

```diff
@@ -26,37 +26,10 @@ class TxHistory extends Component<Props> {
     this.props.txhistory.reset()
   }
 
-  renderTransactionsCell(tx: ObservableTransactionResponse) {
```

Comment:
@goldylucks as far as I could tell - nothing calls this piece of code hence i've removed it.